Tests/js/views/services/plugins/assets/ez-userloadplugin-tests.js

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-userloadplugin-tests', function (Y) {
+    var tests, registerTest,
+        Assert = Y.Assert;
+
+    tests = new Y.Test.Case({
+        name: "eZ User Load Plugin event tests",
+
+        setUp: function () {
+            this.User = function () {
+                this.set = tests._set;
+                this.load = tests._load;
+            };
+
+            this.userId = '/my/user/42';
```

Comment:
this does not need to be global so it's better to keep it in local variable in the test methods.